src/cql/r4/FHIRHelpers.cql

```diff
@@ -6,13 +6,20 @@ define function ToInterval(period FHIR.Period):
     if period is null then
         null
     else
-        Interval[period."start".value, period."end".value]
+        if period."start" is null then
+            Interval(period."start".value, period."end".value]
```

Comment:
Does line 10 have a bracket error?

Reply:
@mldorr, no, the opening parenthesis is intentional, in CQL, when an interval has a null boundary, if the boundary is open, it means the starting point of the interval is unknown, whereas if the boundary is closed (inclusive), it means the starting point of the interval is "ongoing" (i.e. beginning of time). So I'm explicitly using the "unknown" interpretation for the starting boundary because that makes the most sense for what a Period with a null start in FHIR means.